CMFA publish update

woodbe opened the pull request:
This should be ready for wider review. This will close #22, close #23, close #25, close #31, close #32.

All the notes have been removed, and several modifications were done. Individual notes were added in each ticket. Some were also added in #19

Commit:
Updated ESR

This is an attempt to be a "final" version of the ESR that could be sent for wider review.

Commit:
Revert "Updated ESR"

This reverts commit d8319bd745c43495ca21fa22b8d5aa39c1f6800b.

Commit:
Update for wider review

This is hopefully sufficient for wider review.

n-kai reviewed (changes requested):
Note under "Optional Extensions" and "Outside the Scope of Evaluation" should be removed because they are additional text only for iTC members.

Commit:
removal of notes

to clear out missed internal notes

n-kai committed:
Update CMFA_ESR.adoc

Fix typos and replace "verification" with "biometric verification" because term "verification" are used to express signal verification and biometric verification and it could confuse readers.

n-kai reviewed (changes requested):
I created PR #37 to make minor updates. I don't have any other comments on your draft.

Commit:
Merge pull request #38 from biometricITC/n-kai-patch-CMFA

Update CMFA_ESR.adoc

Commit:
re-add original ESR

I had accidentally deleted the original ESR (I wasn't thinking about how the merge to the main branch would work), so this adds the current version back so there won't be a conflict on the final approval.

woodbe commented:
Note that I added back the current BIO ESR here, so it looks like there are more files included, but I had deleted the ESR from the working branch (my mistake), so I have just added it back here so it will be included in the branch when it is merged into the published branch of files. You don't need to review them, they are copies I made today from the main branch.

gfiumara commented on CMFA_ESR.adoc:
```diff
@@ -103,78 +96,46 @@ The configuration process for CMFA is likely to encompass multiple steps, coveri
 CMFA is used primarily for continuous authentication of a user for computers such as smartphones, where the confidence rating is used to determine the locked state of the computer. 
 
-This confidence rating can also be provided to external services such as a PC login at the office, building or room entrance control or ATMs. Those external services may communicate CMFA to the computer so that the external services can request additional user data, such as specific sensor information, to perform secondary CMFA analysis before granting the access to a user.
+In addition to being used by the computer for determining the locked state, the confidence rating can also be provided other systems, such as applications installed on the computer or to external services such as a PC login at the office, building or room entrance control or ATMs. Those external services may communicate CMFA related information to the computer so that the external services can re ...
```
Missing "to"

```suggestion
In addition to being used by the computer for determining the locked state, the confidence rating can also be provided to other systems, such as applications installed on the computer or to external services such as a PC login at the office, building or room entrance control or ATMs. Those external services may communicate CMFA related information to the computer so that the external services can request additional user data, such as specific sensor information, to perform secondary CMFA analysis before granting the access to a user.
```

gfiumara commented on BIO_ESR.adoc:
```diff
@@ -0,0 +1,139 @@
+== Status
+The CCDB Working Group has been requested to develop an Essential Security Requirements (ESR) for biometrics product. The ESR Version 1.0 contained material that was provided by the initiator, IPA (Japan), for a cPP and reviewed by Working Group members (AISEP (Australia), CCN (Spain) and TSE (Turkey)) and initial Biometric Security iTC members (AIST (Japan), Apple (United States), Epoche & Espri ...
+
+Security requirements in this ESR don't depend on biometric characteristics (e.g. fingerprint, face and vein). Therefore the cPP that is to be developed based on this ESR does not depend on biometric characteristics either. The evaluation methodology is defined and described in the supporting document that provides evaluation guidance in common for all biometric characteristics. The supporting do ...
```
```suggestion
Security requirements in this ESR don't depend on biometric characteristics (e.g. fingerprint, face and vein). Therefore the cPP that is to be developed based on this ESR does not depend on biometric characteristics either. The evaluation methodology is defined and described in the supporting document that provides evaluation guidance in common for all biometric characteristics. The supporting document also provides specific evaluation guidance for major biometric characteristics (face, eye, fingerprint and finger/palm vein).
```

Commit:
Update CMFA_ESR.adoc

Co-authored-by: Greg Fiumara <gregory.fiumara@nist.gov>

Commit:
Update BIO_ESR.adoc

Co-authored-by: Greg Fiumara <gregory.fiumara@nist.gov>

Commit:
Update CMFA_ESR.adoc

changes from the call
